cassdegrees/templates/widgets/staff/rulescripts.html

```diff
@@ -81,6 +84,28 @@
         </div>
         <div v-else v-html="info_msg"></div>
 
+        <div class="modal" v-if="show_help">
+            <div class="modal-background" v-on:click="show_help = false"></div>
+            <div class="modal-card">
+                <div class="card">
+                    <header class="box-header">
+                        Help for Subplan Description Field
+                    </header>
+                    <div class="box-solid box-has-footer">
+                        <p>
+                            This description field can be used to add more context to a subplan requirement. For example,
+                            you may simply wish to specify that students must pick "one Arts Major", or you could add
+                            additional criteria by saying students must pick "one Arts Major which has a different
+                            name to their chosen Minor".
```

Comment:
Perhaps describe what contexts this will be used in (PDF)?

Reply:
Good thought! I've added some extra context.